pkgs/development/tools/database/sqlitebrowser/default.nix

```diff
@@ -1,32 +1,42 @@
-{ mkDerivation, lib, fetchFromGitHub, qtbase, qttools, sqlite, cmake }:
+{ mkDerivation, lib, fetchFromGitHub, cmake, antlr
+, qtbase, qttools, qscintilla, sqlite }:
 
-mkDerivation rec {
-  version = "3.10.0";
+let
+  qscintillaLib = (qscintilla.override { withQt5 = true; });
+
+in mkDerivation rec {
+  version = "3.10.1";
   name = "sqlitebrowser-${version}";
 
   src = fetchFromGitHub {
     repo   = "sqlitebrowser";
     owner  = "sqlitebrowser";
     rev    = "v${version}";
-    sha256 = "1fwr7p4b6glc3s0a06i7cg8l9p1mrcm4vyhyf2wi89cyg22rrf5c";
+    sha256 = "1brzam8yv6sbdmbqsp7vglhd6wlx49g2ap8llr271zrkld4k3kar";
   };
 
-  buildInputs = [ qtbase qttools sqlite ];
-  nativeBuildInputs = [ cmake ];
+  buildInputs = [ qtbase qscintillaLib sqlite ];
+
+  nativeBuildInputs = [ cmake antlr qttools ];
 
-  cmakeFlags = [ "-DUSE_QT5=TRUE" ];
+  enableParallelBuilding = true;
 
-  # A regression was introduced in CMakeLists.txt on v3.9.x
-  # See https://github.com/sqlitebrowser/sqlitebrowser/issues/832 and issues/755
+  # We have to patch out Test and PrintSupport to make this work with Qt 5.9
+  # It can go when the application supports 5.9
   postPatch = ''
-    substituteInPlace CMakeLists.txt --replace 'project("DB Browser for SQLite")' 'project(sqlitebrowser)'
+    substituteInPlace CMakeLists.txt \
+      --replace Test         "" \
+      --replace PrintSupport ""
+
+    substituteInPlace libs/qcustomplot-source/CMakeLists.txt \
+      --replace PrintSupport ""
   '';
 
   meta = with lib; {
     description = "DB Browser for SQLite";
-    homepage = http://sqlitebrowser.org/;
-    license = licenses.gpl3;
-    maintainers = [ maintainers.matthiasbeyer ];
-    platforms = platforms.linux; # can only test on linux
+    homepage    = http://sqlitebrowser.org/;
+    license     = licenses.gpl3;
+    maintainers = with maintainers; [ matthiasbeyer ];
+    platforms   = platforms.linux; # can only test on linux
```

Comment:
Please, don't align attributes for simplicity and consistency with other packages.

Reply:
We need a style guide...